apps/ledger-live-desktop/src/renderer/hooks/useDateFormatter.ts

```diff
@@ -0,0 +1,69 @@
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { localeSelector } from "~/renderer/reducers/settings";
+
+/**
+ *
+ * @param d1 date one
+ * @param d2 date two
+ * @returns true if dates are equals, else returns false.
+ */
+
+function dateEq(d1: Date, d2: Date) {
+  return (
+    d1.getFullYear() === d2.getFullYear() &&
+    d1.getMonth() === d2.getMonth() &&
+    d1.getDate() === d2.getDate()
+  );
+}
+
+/**
+ *
+ * @param from date from which to execute the function. Default: today.
+ * @returns an object containing 3 dates relatives to `from`: yesterday, today and tomorrow.
+ */
+const getDatesAround = (from: Date = new Date()) => {
+  const today = from;
```

Comment:
I think that you want to do const today = new Date(), no?
Otherwise, from my understanding, you will always display "Today" as it's computed from the date you send as parameter, am I wrong ?